data/campaigns/Heir_To_The_Throne/scenarios/13_The_Dwarven_Doors.cfg

```diff
@@ -160,7 +160,78 @@
         #
         {VARIABLE_OP true_entrance_location rand "1..2"}
 
-        # TODO: manually add snow detritus
+        [lua]
+            code=<<
+
+local wml_actions = wesnoth.wml_actions
```

Comment:
Makes sense.

I do wonder if there's any point in defining a WML tag for a one-use operation - the contents of that function could just be moved to global scope and use wesnoth.get_variable to determine the correct coverage... but I guess it doesn't really matter.